### DISCUSSION TITLE
See Temp log Graph over a time period

### DISCUSSION THREAD

**Discussion by katastrophe92** [Rem0o/FanControl.Releases#2009] [Q&A]
Hello y'all! Is there a way to have a temp logs graph over a period of time or a way to see sensors temperature logs over a period of time? Like for example: Min, Max, Average temp in 30 min of time.
Hope someone know what I mean 🤣  Thanks!

**Comment by NazZaR** [comment]
I would love to see fan speed charts as well, to see those unwanted fluctuations.

**Comment by noalear** [comment]
I third this request. I would love to be able to log temps and rpms right in this application. 

It would really help me figure out where the my oscillations are coming from.

**Comment by dev-mindz** [comment]
Came here to ask for this as well. Been using HWiNFO for this, but i think there is some conflict after prolonged use. CPU temps stop updating after a while(both in HWiNFO and FanControl). Does not happen when i'm not using HWiNFO.

Current setup for tuning the fans
![Screenshot 2025-02-07 162957](https://github.com/user-attachments/assets/1a84ab79-dd43-430a-8251-5663a783e232)
